packages/cli/src/bin/helpers.ts

```diff
@@ -0,0 +1,10 @@
+import path from 'path';
+import findUp from 'find-up';
+
+export default function findRootDirectory(cwd: string): string | null {
+  const filePath = findUp.sync(['package.json', 'zos.json'], { cwd });
+  if (!filePath) return null;
+  const dir = filePath.split(path.sep);
```

Comment:
My two cents.
There is a method [path.dirname](https://nodejs.org/api/path.html#path_path_dirname_path) so you can do `return path.dirname(filePath);`. Feels more natural.